R/content.R

```diff
@@ -1587,21 +1581,38 @@ get_content_packages <- function(content) {
 search_content <- function(
   client,
   q = NULL,
-  page_number = 1,
-  page_size = 500,
```

Comment:
I figured we don't need to expose these to users, since pagination is kind of an implementation detail here.

Reply:
> But retaining them and passing them down might make it easier to test performance under different values

Yeah, that's true. A few thoughts: (1) one _should be able to_ pass these down in `...` even if they are not included in the signature explicitly. (2) what do our other functions do? If they expose these explicitly so that it's obvious there are nobs to tune here, I'm fine including them. But also fine to not (so long as someone can pass them in `...`)